netket/_vmc.py

```diff
@@ -11,6 +11,7 @@
 
 from netket.vmc_common import info, tree_map
 from netket.abstract_variational_driver import AbstractVariationalDriver
+import jax
```

Comment:
jax should never be imported directly but only if available, see how it is imported in other modules through netket.utils.has_jax